### ISSUE TITLE
new odo workflow: odo component create --context

### ISSUE THREAD

**Issue by kadel** [openshift/odo#1372]
Add `--context` flag to `odo create` command.

Context specifies the context / directory where `.odo` config directory is.

The default value for `--context` is the current directory.

`odo component create` creates local config for component in  `./.odo/` directory.
`--context` flag specifies where the `.odo` should be created
context directory needs to exit, otherwise error out

this flag will replace currently existing`--local`

example:

```
odo component create 
```
config in `./.odo/`

```
odo component create --context /my/project 
```
config in `/my/project/.odo/`

related to #1366

**Comment by kadel** [comment]
@anmolbabu this was implemented in #1360 right?

**Comment by anmolbabu** [comment]
Yeah I'll quickly close it
